src/main/app/forms/models/defendantRepresented.ts

```diff
@@ -0,0 +1,53 @@
+import { IsDefined, IsIn, MaxLength, ValidateIf } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { YesNo } from 'forms/models/yesNo'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+
+export class ValidationErrors {
+  static readonly DEFENDANT_IS_REPRESENTED_REQUIRED: string = 'Choose yes if defendant is represented'
+  static readonly COMPANY_NAME_REQUIRED: string = 'Enter your company name'
```

Comment:
this should be: 'Enter the legal representative’s company name' as state in the validation document